docs/tooling/cli-guides/setup-a-devnet.md

```diff
@@ -758,3 +758,144 @@ curl http://52.204.202.216:9650/ext/bc/JjDfmxM3hAEX3VuaKH4PpQskhrvp2pzGTgYLpDwin
 
 C-Chain endpoint follows the same scheme but with the blockchain alias of `C`: `http://52.204.202.216:9650/ext/bc/C/rpc`
 
+<!-- vale off -->
+
+#### Deploy Teleporter
+
+<!-- vale on -->
```

Comment:
is this needed?

Reply:
Vale was flagging "Teleporter" as a typo, so unless there's a way to apply custom rules to an entire file or repo, this is necessary.